### DISCUSSION TITLE
Installing 2.7.7 Asks to install dfacc.msi--> What is it ?

### DISCUSSION THREAD

**Discussion by KT-Andi** [keepassxreboot/keepassxc#10492] [Q&A] [answered]
Hi,

Started the Update of KeepassXC from 2.7.6 to 2.7.7 .
The installer asks to install **dfacc.msi**.
Google does not find anything about this.
Can you please explain what this installer is for and if it's ok to proceed with ? 

Greetings
Andreas

**Comment by droidmonkey** [comment]
When do you see that during the install?  Did you get the installer from our downloads?

**Comment by KT-Andi** [comment]
I did get the installer from: **https://keepassxc.org/download/#windows**

Order of steps:

![image](https://github.com/keepassxreboot/keepassxc/assets/106165727/ec24b3cc-1dce-4c3d-9bc5-3b7c36672b31)
![image](https://github.com/keepassxreboot/keepassxc/assets/106165727/e4732449-81de-4765-8c3f-0583697263a3)
![image](https://github.com/keepassxreboot/keepassxc/assets/106165727/3d128593-2cbe-4b55-b70d-ff0eec8459de)
![image](https://github.com/keepassxreboot/keepassxc/assets/106165727/789d4739-76be-4633-b121-aea4aa13ec1f)

The dfacc.msi Popup was when trying at home, today i tried in the office, now the popup asks for  e17c11.msi

![image](https://github.com/keepassxreboot/keepassxc/assets/106165727/d8a2153c-0233-429a-8324-dc9b1c9df388)

**Reply by KT-Andi** [reply]
...just recognized in the picture, did the certificate expire (15.03.2024) ?

**Reply by droidmonkey** [reply, marked as answer]
That is just the internal installer file that is a normal behavior. The important thing is that it is signed and the signature is valid by Windows. It is, since the banner color is blue. The certificate is expired, but the signature is still valid.

**Reply by KT-Andi** [reply]
OK, Thanks for the confirmation. 👍
